doc/source/csv.rst

```diff
@@ -0,0 +1,23 @@
+==============
+ CSV Samples
+==============
+
```

Comment:
Similar to the comment I left on the XML addition, I think we're going to want to make clear what the type of "data" will be, since it changes from a list of dicts (when they indicate there are headers) to a list of tuples (when that flag is false).

Reply:
Added minimal documentation like for XML.